package-lock.json

```diff
@@ -1,3065 +1,8 @@
 {
   "name": "slava-ukraine",
   "version": "0.1.0",
-  "lockfileVersion": 2,
```

Comment:
I used `npm@8`, you're probably using 6th. Time to update :)

Reply:
time to use yarn ;) but ok